biome.json

```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.6.3/schema.json",
+	"files": {
+		"ignore": ["**/*.astro"]
```

Comment:
Biome currently errors on top-level returns in astro files, and there's no clean workaround, so it's disabled there for now